migrations/4_deploy_plcr.js

```diff
@@ -30,8 +30,8 @@ module.exports = (deployer, network, accounts) => {
 
```

Comment:
should remove these changes too since it's out of scope of the abstraction PR, even though they make sense